builder/internals.go

```diff
@@ -174,8 +175,7 @@ func (b *Builder) runContextCommand(args []string, allowRemote bool, allowDecomp
 	if len(copyInfos) == 0 {
 		return fmt.Errorf("No source files were specified")
 	}
-
-	if len(copyInfos) > 1 && !strings.HasSuffix(dest, "/") {
+	if len(copyInfos) > 1 && !strings.HasSuffix(dest, string(os.PathSeparator)) {
```

Comment:
Since 'dest' comes from the Dockerfile and given what you said about how people are supposed to use '/' in the Dockerfile should this remain '/' instead of '\\' when the daemon is windows?